src/IECoreScene/bindings/ShaderNetworkAlgoBinding.cpp

```diff
@@ -50,7 +60,7 @@ void IECoreSceneModule::bindShaderNetworkAlgo()
 
 	def( "addShaders", &ShaderNetworkAlgo::addShaders, ( arg( "network" ), arg( "sourceNetwork" ), arg( "connections" ) = true ) );
 	def( "removeUnusedShaders", &ShaderNetworkAlgo::removeUnusedShaders );
-	def( "convertOSLComponentConnections", &ShaderNetworkAlgo::convertOSLComponentConnections );
+	def( "convertOSLComponentConnections", &convertOSLComponentConnectionsWrapper, ( arg( "network" ), arg( "oslVersion" ) = 10900 ) );
```

Comment:
FWIW, I don't know of any uses of the python binding outside of the Cortex tests, so you could drop the default value here to specifying the version properly

Reply:
As long as I'm providing backwards compatibility on the C++ side, we may as well keep it on the Python side.